Assets/Scripts/Interactions/IInteractor.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface IInteractor
+{
+    void HandleInteraction(IInteractable interactable);
+}
```

Comment:
if I'm not wrong we wanted ICanInteract LOL. And also, the interface should probably be empty

Reply:
because it seems as though this is a direct circular dependency

Reply:
Ok sures will fix it!